cfme/tests/services/test_dialog_regex_validation_in_catalog.py

```diff
@@ -169,4 +169,54 @@ def test_regex_dialog_validation_error():
             5.
             6. It shouldn't gives the validation error
     """
-    pass
+    service_dialog = appliance.collections.service_dialogs
+    element_data = {
+        "element_information": {
+            "ele_label": fauxfactory.gen_alphanumeric(15, start="ele_label_"),
+            "ele_name": fauxfactory.gen_alphanumeric(15, start="ele_name_"),
+            "ele_desc": fauxfactory.gen_alphanumeric(15, start="ele_desc_"),
+            "choose_type": "Text Box",
+        },
+        "options": {"validation_switch": True, "validation": "^[0-9]*$"},
+    }
+    sd = service_dialog.create(
+        label=fauxfactory.gen_alphanumeric(start="dialog_"), description="my dialog"
+    )
+    tab = sd.tabs.create(
+        tab_label=fauxfactory.gen_alphanumeric(start="tab_"), tab_desc="my tab desc"
+    )
+    box = tab.boxes.create(
+        box_label=fauxfactory.gen_alphanumeric(start="box_"), box_desc="my box desc"
+    )
+    box.elements.create(element_data=[element_data])
+
+    navigate_to(sd, "Edit")
+    view = appliance.browser.create_view(EditElementView)
+    label = element_data["element_information"]["ele_label"]
+    view.element.edit_element(label)
+    view.options.click()
+    assert view.options.validation_switch.fill(False)
+    view.ele_save_button.click()
+    view.save_button.click()
+
+    catalog_item = appliance.collections.catalog_items.create(
+        appliance.collections.catalog_items.GENERIC,
+        name=fauxfactory.gen_alphanumeric(15, start="cat_item_"),
+        description="my catalog",
+        display_in=True,
+        catalog=catalog,
+        dialog=sd)
+
+    @request.addfinalizer
+    def _cleanup():
+        if catalog_item.exists:
+            catalog_item.delete()
+        sd.delete_if_exists()
+
+    ele_name = element_data["element_information"]["ele_name"]
+    service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
+    view = navigate_to(service_catalogs, 'Order')
+    view.fields(ele_name).fill("a")
```

Comment:
Is there any reason behind filling with "a"?

Reply:
yes, This is under testStep 6